Client/src/Components/Graph/Graph.js

```diff
@@ -51,51 +32,49 @@ datalist.push(dataset3)
 datalist.push(dataset4)
 
 class Graph extends Component {
```

Comment:
Mount your changes after pulling Tiras latest changes

Reply:
Develop branch is now merged into the zoom branch.